### ISSUE TITLE
Notice: Undefined index: all in color_css_alter() line 57 of color.module

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#1941]
I got this message while trying to create a subtheme for Bartik...
- created a folder under `/themes/mybartik`
- created a `mybartik.info` file under that folder with only the minimum info required:
  
  ```
  name = myBartik
  description = my custom theme based on core Bartik.
  type = theme
  backdrop = 1.x
  base theme = bartik
  
  stylesheets[screen][] = css/mystyle.css
  ```
- created my custom style file (only additional things I need to change from the base theme) under `/themes/mybartik/css/mystyle.css`.
- switched the default theme to my new custom theme
- visit home page ->

```
Notice: Undefined index: all in color_css_alter() (line 57 of /var/www/core/modules/color/color.module).
Warning: Invalid argument supplied for foreach() in color_css_alter() (line 57 of /var/www/core/modules/color/color.module).
```

**Comment by klonos** [comment]
Disabling color.module makes the message go away of course, but adding `settings[color] = false` has no effect.

**Comment by hosef** [comment]
This is because the Color module does not support sub-themeing very well.
Currently, if you want to use the color module in a sub theme you have to:
- Copy the color folder from the base theme
- Change the css files list in color.inc to point to your theme's new css files
- Make sure that all the colors that you need to modify are specified in your new styles

The Color module should probably be modified to better support sub themes by pulling in the color settings and modifying css files from the base theme.

**Comment by hosef** [comment]
This should be fixed now that #3348 is done.
